pages/search/index.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from 'react'
+import Button from '@mui/material/Button'
+import Checkbox from '@mui/material/Checkbox'
+import Chip from '@mui/material/Chip'
+import FormControl from '@mui/material/FormControl'
+import InputLabel from '@mui/material/InputLabel'
+import ListSubheader from '@mui/material/ListSubheader'
+import MenuItem from '@mui/material/MenuItem'
+import Select from '@mui/material/Select'
+import Stack from '@mui/material/Stack'
+import TextField from '@mui/material/TextField'
+import Typography from '@mui/material/Typography'
+import {
+  creditTypes,
+  schoolsWhereCategoryCode,
+  schoolFindById,
+} from '/lib/models'
+import Link from 'next/link'
+
+export default function Search() {
+  const [search, setSearch] = useState('')
+
+  const [filters, setFilters] = useState({
+    schools: [],
+    search: '',
+    creditType: '',
+    courseNumbers: '',
+  })
```

Comment:
I think it would be useful to preserve any previous search/filters when navigating back to the Search page.

If I've searched for "Culinary" and then click Search on the top nav bar, I would expect it to continue/modify my previous search, not reset:
<img width="868" alt="Search Results" src="https://user-images.githubusercontent.com/33558504/212929761-0c8c0e13-8450-4030-9f3c-c1abd337e397.png">
<img width="859" alt="Clicking Search Clears Previous Filters" src="https://user-images.githubusercontent.com/33558504/212930042-f38e5ba9-7dc6-4aef-868a-02650631f6ad.png">